test/__snapshots__/context.test.js.snap

```diff
@@ -13,23 +13,3 @@ exports[`no double runs for getDerivedStateFromProps 1`] = `
   Test-content
 </div>
 `;
-
```

Comment:
Why remove these snapshots? Tests are failing because of that.

Reply:
It is very strange 🤔